src/env.ts

```diff
@@ -0,0 +1,9 @@
+export const IS_TEST = process.env.NODE_ENV === 'test'
+export const IS_DEV = __DEV__
+export const IS_PROD = !IS_DEV
+export const LOG_DEBUG = process.env.EXPO_PUBLIC_LOG_DEBUG || ''
+export const LOG_LEVEL = (process.env.EXPO_PUBLIC_LOG_LEVEL || 'info') as
+  | 'debug'
+  | 'info'
+  | 'warn'
+  | 'error'
```

Comment:
I've found a module file to be super helpful with testing since you can mock the variables more easily than globals. Expo automatically replaces `process.env.<var>` calls with the strings.